### PR TITLE
Bump bk version to 4.11.0

### PR DESCRIPTION
**Pull request by sijie**

### CODE REVIEW

File: distribution/server/src/assemble/LICENSE.bin.txt [hunk 1]
```diff
@@ -384,32 +384,32 @@ The Apache Software License, Version 2.0
     - org.apache.logging.log4j-log4j-web-2.10.0.jar
  * Java Native Access JNA -- net.java.dev.jna-jna-4.2.0.jar
  * BookKeeper
-    - org.apache.bookkeeper-bookkeeper-common-4.10.0.jar
```
Comment (rdhabalia) [inline distribution/server/src/assemble/LICENSE.bin.txt:387]: should we document changing version as part of release process?

**Comment by wolfstudy** [comment]
```
src/check-binary-license ./distribution/server/target/apache-pulsar-*-bin.tar.gz
```

Output:

```
org.apache.httpcomponents-httpclient-4.5.5.jar unaccounted for in LICENSE
org.apache.httpcomponents-httpcore-4.4.1.jar unaccounted for in LICENSE
org.apache.thrift-libthrift-0.12.0.jar unaccounted for in LICENSE
org.bouncycastle-bcpkix-jdk15on-1.60.jar unaccounted for in LICENSE
org.bouncycastle-bcprov-ext-jdk15on-1.60.jar unaccounted for in LICENSE
org.bouncycastle-bcprov-jdk15on-1.60.jar unaccounted for in LICENSE
bcpkix-jdk15on-1.60.jar unaccounted for in lib/presto/LICENSE
bcprov-ext-jdk15on-1.60.jar unaccounted for in lib/presto/LICENSE
bcprov-jdk15on-1.60.jar unaccounted for in lib/presto/LICENSE
commons-logging-1.2.jar unaccounted for in lib/presto/LICENSE
httpclient-4.5.5.jar unaccounted for in lib/presto/LICENSE
httpcore-4.4.9.jar unaccounted for in lib/presto/LICENSE
netty-transport-native-epoll-4.1.48.Final.jar mentioned in lib/presto/LICENSE, but not bundled
src/check-binary-license: line 111: echo: write error: Broken pipe
commons-logging-1.1.1.jar mentioned in lib/presto/LICENSE, but not bundled
```

**Comment by codelipenghui** [comment]
Close this issue via #8270

**Pull request closed without merging**